stable/prometheus-cloudwatch-exporter/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.1.0"
 description: A Helm chart for prometheus cloudwatch-exporter
 name: prometheus-cloudwatch-exporter
-version: 0.1.4
+version: 0.1.5
```

Comment:
please bump to a major version

Reply:
also change the image from latest to `cloudwatch_exporter-0.5.0`

Reply:
Done